src/cascadia/TerminalSettingsModel/IInheritable.h

```diff
@@ -201,12 +197,12 @@ private:                                                                    \
         /*iterate through parents to find a value*/                         \
         for (auto parent : _parents)                                        \

```

Comment:
```suggestion
        for (const auto& parent : _parents)                                 \
```

Reply:
idk how we missed this in code review

shouldn't we have had a helper for "walk parents and do a thing"? I thought that was part of the brief.

Reply:
The problem I ran into by taking this out into its own helper method was that we're returning a value and calling a specific `_get##name##Impl()`. Figured it'd be less of a headache to throw this in a macro.

The `const &` thing is my bad tho. Sorry! I think I went through all of the other ones in the PR, but I just forgot to check the macro?